package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "skynet-js",
-  "version": "4.2.0-beta",
+  "version": "4.2.1-beta",
```

Comment:
nit: Do we really need these beta tags? I would much rather not have them.

Reply:
That's useful feedback - what don't you like about them? We're doing a beta for the large file changes, so the community can test it before rolling it out.

Reply:
Resolving so we can merge this.